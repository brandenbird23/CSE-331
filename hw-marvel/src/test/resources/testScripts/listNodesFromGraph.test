# Tests that all nodes are printed from a file

LoadGraph nfl nflPlayers.csv
ListNodes nfl