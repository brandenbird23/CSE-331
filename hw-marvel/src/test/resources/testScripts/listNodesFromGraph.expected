# Tests that all nodes are printed from a file

loaded graph nfl
nfl contains: BROCK-PURDY DANIEL-JONES GENO-SMITH GEORGE-KITTLE JORDYN-BROOKS MIKE-EVANS PEYTON-MANNING ROB-GRONKOWSKI RUSSEL-WILSON SAQUON-BARKLEY TOM-BRADY TRISTAN-WURFS